Automattic/jetpack
# Verbum Comments: Fix iframe resize logic

Fixes https://github.com/Automattic/wp-calypso/issues/92671

## Proposed changes:

* Update resize logic for Verbum Comments iframes. If there are multiple iframes on a page, these changes will ensure resizing Verbum will also update the correct iframe height.

### Other information:


## Jetpack product discussion



## Does this pull request change what data or activity we track or use?



## Testing instructions:






* Apply this patch to your sandbox: D163990-code
* Build and sync the jetpack plugin changes to your atomic site: 
Build command: `jetpack build` -> select `plugins` -> select `jetpack`.
Sync command: `jetpack rsync jetpack`
* Build and sync the jetpack-mu-plugin changes to your sandbox:
Build and sync command ` pnpm build-js && jetpack rsync mu-wpcom-plugin`
* Go the site editor (edit template) for your atomic site and add a `query loop` with a `comment form` component to your single post page, so youre able to see two comment forms on your site.

<img width="1798" alt="CleanShot 2024-10-16 at 16 17 47@2x" src="https://github.com/user-attachments/assets/bafcb216-d976-48d4-8cd0-9f5c48679e85">

* Resize both comment section by adding text or simply clicking enter to add a new line.
* Verify that the iframes are resizing correctly.

https://github.com/user-attachments/assets/8b1740d9-0dfe-4d8e-889c-1bcf6385b6c7
